lib/init.js

```diff
@@ -5,47 +5,30 @@ import os from 'os';
 import tmp from 'tmp';
 import {CompositeDisposable} from 'atom';
 
-export default {
-  config: {
-    executablePath: {
-      type: 'string',
-      default: ''
-    },
-    includeDirs: {
-      type: 'array',
-      default: [],
-      items: {
-        type: 'string'
-      }
-    },
-    pa: {
-      type: 'array',
-      default: [],
-      items: {
-        type: 'string'
-      }
-    }
-  },
+let executablePath;
```

Comment:
These should stay as `this.` properties. The reason some providers don't currently do that was due to a bug in how the `lint` function was being (improperly) declared which prevented it from working properly.